From 7b2936a6d6736496ccf7e2db5c8365d7043ee732 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Mon, 3 Sep 2018 20:25:29 +0200
Subject: [PATCH] Fix autotests with -DBUILD_QCH:BOOL=TRUE

Summary:
After 507415c54bd111fbb35716bd9809119d990f9a16 KF5I18nQchTargets.cmake
needs similar adjustments.

Test Plan: The test passes again, KF5I18nQchTargets is still installed

Subscribers: kde-frameworks-devel

Tags: #frameworks

Differential Revision: https://phabricator.kde.org/D15251
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f04c3e5..c1097fb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,7 +88,7 @@ set(CMAKECONFIG_INSTALL_DIR "${KDE_INSTALL_CMAKEPACKAGEDIR}/KF5I18n")
 if (BUILD_QCH)
     ecm_install_qch_export(
         TARGETS KF5I18n_QCH
-        FILE KF5I18nQchTargets.cmake
+        FILE "${CMAKE_BINARY_DIR}/cmake/KF5I18nQchTargets.cmake"
         DESTINATION "${CMAKECONFIG_INSTALL_DIR}"
         COMPONENT Devel
     )
